.gitignore

```diff
@@ -251,5 +268,22 @@ paket-files/
 .idea/
 *.sln.iml
 
-# Ignore customized Config.json
-Source/ACE/Config.json
```

Comment:
we lost this in the in the change, we would still want to ignore this file, yeah?